src/server/scripts/ExilesReach/zone_exiles_reach.cpp

```diff
@@ -2186,6 +2188,1317 @@ struct areatrigger_find_the_lost_expedition_follower : AreaTriggerAI
+                        me->CastSpell(me, SPELL_RANGED_ROOT_DNT);
+                        _senondaryCheck = true;
+                        break;
+                    case CLASS_PALADIN:
+                        ConversationWithPlayer(CONVERSATION_READY_COMBAT);
+                        _rightiousCheck = true;
+                        _senondaryCheck = true;
+                        break;
+                    case CLASS_ROGUE:
+                        ConversationWithPlayer(CONVERSATION_READY_COMBAT);
+                        _comboPointCheck = true;
+                        _senondaryCheck = true;
+                        break;
+                    case CLASS_PRIEST:
+                        ConversationWithPlayer(CONVERSATION_READY_COMBAT);
+                        me->CastSpell(me, SPELL_AGGRO_RADIUS_CHECK_DNT_PRIEST);
+                        me->CastSpell(me, SPELL_RANGED_ROOT_DNT);
+                        _senondaryCheck = true;
+                        break;
+                    case CLASS_SHAMAN:
+                        ConversationWithPlayer(CONVERSATION_READY_COMBAT);
+                        me->CastSpell(me, SPELL_AGGRO_RADIUS_CHECK_DNT_SHAMAN);
+                        me->CastSpell(me, SPELL_RANGED_ROOT_DNT);
+                        _senondaryCheck = true;
+                        break;
+                    case CLASS_MAGE:
+                        ConversationWithPlayer(CONVERSATION_READY_COMBAT);
+                        me->CastSpell(me, SPELL_AGGRO_RADIUS_CHECK_DNT_WARRIOR_MAGE);
+                        me->CastSpell(me, SPELL_RANGED_ROOT_DNT);
+                        _senondaryCheck = true;
+                        break;
+                    case CLASS_WARLOCK:
+                        ConversationWithPlayer(CONVERSATION_READY_COMBAT);
+                        me->CastSpell(me, SPELL_AGGRO_RADIUS_CHECK_DNT_WARLOCK);
+                        me->CastSpell(me, SPELL_RANGED_ROOT_DNT);
+                        _senondaryCheck = true;
+                        break;
+                    case CLASS_DRUID:
+                        ConversationWithPlayer(CONVERSATION_READY_COMBAT);
+                        me->CastSpell(me, SPELL_AGGRO_RADIUS_CHECK_DNT_DRUID);
+                        me->CastSpell(me, SPELL_RANGED_ROOT_DNT);
+                        _senondaryCheck = true;
+                        break;
+                    case CLASS_MONK:
+                        ConversationWithPlayer(CONVERSATION_READY_COMBAT);
+                        break;
+                    default:
+                        break;
+                }
+                break;
+            }
+            default:
+                break;
+            }
+    }
+
+    void DamageTaken(Unit* /*attacker*/, uint32& damage, DamageEffectType /*damageType*/, SpellInfo const* /*spellInfo = nullptr*/) override
+    {
+        if (me->GetHealth() <= damage)
+        {
+            damage = 0;
+            me->SetHealth(1);
```

Comment:
damage = me->GetHealth - 1;

This makes the SetHealth hack unneeded and negates any damage beyond the 1 hp threshold

Reply:
yeah i'll remove that